### PR TITLE
variety: fixed GLib-GIO-ERROR error

### PR DESCRIPTION
**Pull request by p3psi-boo**
###### Motivation for this change
Related issue: https://github.com/NixOS/nixpkgs/issues/82387

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/variety/default.nix [hunk 1]
```diff
@@ -27,10 +28,12 @@ buildPythonApplication rec {
 
   nativeBuildInputs = [ intltool wrapGAppsHook ];
 
-  buildInputs = [ distutils_extra ];
+  buildInputs = [ makeWrapper distutils_extra ];
 
   doCheck = false;
 
+  postInstall = "wrapProgram $out/bin/variety --suffix XDG_DATA_DIRS : ${gtk3}/share/gsettings-schemas/${gtk3.name}/";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/variety/default.nix:35]:
```suggestion
  postInstall = ''
    wrapProgram $out/bin/variety --suffix XDG_DATA_DIRS : ${gtk3}/share/gsettings-schemas/${gtk3.name}/
  '';
```

File: pkgs/applications/misc/variety/default.nix [hunk 2]
```diff
@@ -10,6 +10,7 @@
 , hicolor-icon-theme
 , librsvg
 , wrapGAppsHook
+, makeWrapper
 }:
 
 with python37Packages;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/variety/default.nix:16]: Can we use python3 here?

**Review by workflow — APPROVED** [review]
This works great and fixes `variety` for me on i3.

Thank you!

File: pkgs/applications/misc/variety/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/applications/misc/variety/default.nix:35]: @p3psi-boo ping :)

**Pull request merged**